docu/sphinx/source/conf.py

```diff
@@ -36,8 +36,8 @@ def getDocRoot():
 # -- Project information -----------------------------------------------------
 
 project = u'Igor Unit Testing Framework'
-copyright = u'2018, Thomas Braun, Michael Huth, Matthias Kastner'
-author = u'Thomas Braun, Michael Huth, Matthias Kastner'
+copyright = u'2012-2018 All Contributors'
+author = u'All Contributors'
```

Comment:
If it is written in this way, there is no additional information in, removing it would be equally possible.
I would rather drop that commit.